src/js/tabs/cli.js

```diff
@@ -152,19 +164,23 @@ TABS.cli.read = function (readInfo) {
         MAC only understands CR
         Linux and Unix only understand LF
         Windows understands (both) CRLF
-        Chrome OS currenty unknown
+        Chrome OS currently unknown
     */
     var data = new Uint8Array(readInfo.data),
-        text = "";
+        cliOutput = "",
+        cliPromptText = "",
+        currentLine = "";
 
     for (var i = 0; i < data.length; i++) {
+        const currentChar = String.fromCharCode(data[i]);
         if (CONFIGURATOR.cliValid) {
-            if (data[i] == 27 || this.sequenceElements > 0) { // ESC + other
+            const escapeSequenceCode = 27;
+            if (data[i] == escapeSequenceCode || this.sequenceElements > 0) { // ESC + other
                 this.sequenceElements++;
 
                 // delete previous space
                 if (this.sequenceElements == 1) {
-                    text = text.substring(0, text.length -1);
+                    cliOutput = text.substring(0, text.length - 1);
```

Comment:
`text` here is causing ReferenceError.

Reply:
Good spot, I’ll fix when I have laptop access.

I restored this last minute. Admittedly couldn’t see what it’s purpose was, do you know?

Also, is there any appetite for automated tests for the configurator? I’d be happy to add some for this file.

Reply:
I don't know what it exactly does, no.

And yes, tests are very welcome. You can add the test runner to `package.json`.